net/wg-installer/wg-client/lib/rpcd_ubus.sh

```diff
@@ -0,0 +1,134 @@
+. /usr/share/libubox/jshn.sh
+
+function query_gw {
```

Comment:
function? this is not bash.

Reply:
You can do both? If u like I can switch to the c-style notation.

Reply:
I'm pretty sure shellcheck complains on this. function is a bashism IIRC.

Reply:
I changed that. Give me some time to run-test again. ;) No access to OpenWrt router for 1 hour.